fields/types/gcsavatar/GcsAvatarField.js

```diff
@@ -0,0 +1,147 @@
+import Field from '../Field';
+import React from 'react';
+import ReactDOM from 'react-dom';
+import get from 'lodash/get';
+import { Button, FormField, FormNote } from 'elemental';
+
+const _ = {
+  get,
+};
+
+const storageDefaults = {
+  scheme: 'https',
+  hostname: 'storage.googleapis.com',
+};
+
+const supportTypes = ['image/gif', 'image/png', 'image/jpeg', 'image/x-tiff', 'image/svg+xml', 'image/bmp'];
```

Comment:
Do we need to support so many file extensions?

Reply:
I'm not sure how many of them should be supported therefore they're identical to the ones in GcsImageField.js.

I'll remove 'image/gif', 'image/x-tiff', 'image/bmp' in next updates as they are not so common in avatars use case.

Reply:
Sure!